pkg/tfvars/openstack/rhcos_image.go

```diff
@@ -110,6 +73,52 @@ func uploadBaseImage(cloud string, localFilePath string, imageName string, clust
 	return nil
 }
 
+func importImage(imageName, imageID string, conn *gophercloud.ServiceClient, f *os.File) error {
+	logrus.Debugf("Using Image Import API to upload RHCOS to the image %q with ID %q", imageName, imageID)
+	stageRes := imagedata.Stage(conn, imageID, f)
+	if stageRes.Err != nil {
+		logrus.Debugf("Image uploading failed")
+		return stageRes.Err
+	}
+	logrus.Debugf("The data was uploaded")
```

Comment:
I think you mean, "image uploaded"  or what is data? 
Just take out the log message an only report a failure I think.